compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -1439,17 +1439,32 @@ mod signal_handler {
 
     extern "C" fn print_stack_trace(_: libc::c_int) {
```

Comment:
can we move this module into separate files?

Reply:
Done~